schema.sql

```diff
@@ -10,4 +10,26 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals
-ADD COLUMN species varchar(20);
\ No newline at end of file
+ADD COLUMN species varchar(20);
+
+CREATE TABLE owners (
+	id INT NOT NULL GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	full_name VARCHAR(20),
```

Comment:
- [ OPTIONAL] I like the fact you try to set a limitation here. But let's assume an owner full name exceeds 20 characters in total. In that case, your database table won't be able to accommodate that owner, right? Therefore, it will be great if you could use 255 for full names. Because normally some full names are longer and you database will accommodate more.